copis/gui/panels/properties.py

```diff
@@ -617,13 +617,13 @@ def on_remote_usb_radio_group(self, event: wx.CommandEvent) -> None:
             self.ptp_rbh.Value = False
             self.ptp_rbh.Disable()
 
-            if self.parent.c.edsdk_enabled:
-                self.parent.c.terminate_edsdk()
+            if self.parent.core.edsdk_enabled:
```

Comment:
Yeah I guess all these `self.parent.core`s can be `self.core`

Reply:
See above.